src/pages/certificates/CertificateGenerate.tsx

```diff
@@ -55,97 +54,123 @@ const CertificateGenerate: FC = () => {
-                <Button
-                  className="u-certificate-button"
-                  onClick={createCert}
-                  appearance="positive"
-                  disabled={isGenerating || certs !== null}
-                  hasIcon={isGenerating}
-                  aria-label={`${
-                    isGenerating ? "Generating" : "Generate"
-                  } certificate`}
-                >
-                  {isGenerating && (
-                    <Icon
-                      className="is-light u-animation--spin"
-                      name="spinner"
-                    />
-                  )}
-                  <span>{isGenerating ? "Generating" : "Generate"}</span>
-                </Button>
-              </Col>
-            </Row>
-          </li>
-          <li className="p-stepped-list__item">
-            <Row>
-              <Col size={8}>
-                <h3 className="p-stepped-list__title">Trust</h3>
-                <div className="p-stepped-list__content">
-                  Download <code>lxd-ui.crt</code> and add it to the LXD trust
-                  store
-                  <div className="p-code-snippet">
-                    <pre className="p-code-snippet__block--icon">
-                      <code>lxc config trust add Downloads/lxd-ui.crt</code>{" "}
-                    </pre>
-                  </div>
-                </div>
-              </Col>
-              <Col size={4}>
-                {certs && (
-                  <Button
-                    className="u-certificate-button"
-                    onClick={() => downloadText("lxd-ui.crt", certs.crt)}
-                  >
-                    Download crt
-                  </Button>
-                )}
-              </Col>
-            </Row>
-          </li>
-          <li className="p-stepped-list__item">
-            <Row>
-              <Col size={8}>
-                <h3 className="p-stepped-list__title">Import</h3>
-                <div className="p-stepped-list__content">
-                  <p>
-                    Download <code>lxd-ui.pfx</code> and import it into your
-                    browser.
-                  </p>
-                  <BrowserImport />
-                </div>
-              </Col>
-              <Col size={4}>
-                {certs && (
-                  <Button
-                    className="u-certificate-button"
-                    onClick={() => downloadBase64("lxd-ui.pfx", certs.pfx)}
-                  >
-                    Download pfx
-                  </Button>
-                )}
-              </Col>
-            </Row>
-          </li>
-          <li className="p-stepped-list__item">
-            <h3 className="p-stepped-list__title">
-              All done <Icon name="success" />
-            </h3>
-          </li>
-        </ol>
-      </Row>
-    </BaseLayout>
+    <main className="l-main certificate-generate">
+      <div className="p-panel">
+        <div className="p-panel__header">
```

Comment:
If you add the vanilla `is-sticky` class here, you'd need to set a CSS rule only for the white background on this header.

Reply:
Did that.
We still need the z-index bump, because the tab underlines should not overlay the header